scss/foundation/components/_tooltips.scss

```diff
@@ -74,11 +74,11 @@ $tooltip-max-width: 300px !default;
         position: absolute;
         top: -($tooltip-pip-size * 2);
         width: 0;
-        #{$default-float}: $tooltip-pip-size;
```

Comment:
Why do we set the tooltip position based on $default-float value? the positioning is handled in the js file.